koku/api/report/queries.py

```diff
@@ -1075,6 +1078,11 @@ def _ranked_list(self, data_list, ranks):
         # Cross join ranks and days to get each field/rank for every day in th query
         ranks_by_day = rank_data_frame.merge(day_data_frame, how="cross")
 
+        # add the ranking columns if they still exists in both dataframes
+        rank_fields.intersection_update(set(ranks_by_day.columns.intersection(data_frame.columns)))
```

Comment:
`set` methods accept an iterable and don't require a `set`. The `pandas.core.indexes.numeric.Int64Index` returned by the `intersection` method is iterable.
```suggestion
        rank_fields.intersection_update(ranks_by_day.columns.intersection(data_frame.columns))
```